scripts/install-aica-docker.sh

```diff
@@ -1,10 +1,56 @@
 #!/bin/bash
 
+function get_shell_rc_path () {
+  local user home_dir
+  user=$(logname)
+  home_dir=$(eval echo "~${user}")
+  if [[ "${SHELL##*|}" == *"zsh"* ]]; then
+    echo "${home_dir}/.zshrc"
+  elif [[ "${SHELL##*|}" == *"bash"* ]]; then
+    if [[ "$OSTYPE" != "darwin"* ]]; then
+      echo "${home_dir}/.bashrc"
+    else
+      echo "${home_dir}/.bashrc"
+    fi
+  else
+    echo "Currently, only zsh and bash shells are supported for autocompletion with aica-docker."
+    exit 1
+  fi
+}
+
+function source_completion_script () {
+  if grep -Rq "/src/aica-docker-completion.sh" "$1"; then
+    echo "Updating auto completion in $1"
+    grep -v /src/aica-docker-completion.sh "$1" > tmpfile && mv tmpfile "$1"
+  else
+    echo "Writing auto completion to $1"
+  fi
+  echo "source ${SCRIPT_DIR}/src/aica-docker-completion.sh" >> "$1"
```

Comment:
For zshrc, you will need to add the following lines
```shell
autoload bashcompinit
bashcompinit
source ${SCRIPT_DIR}/src/aica-docker-completion.sh
```

I tested it like this and it works beautifully